pkg/internal/apis/acme/types_issuer.go

```diff
@@ -221,8 +222,8 @@ type ACMEChallengeSolverHTTP01Ingress struct {
 }
 
 type ACMEChallengeSolverHTTP01GatewayHTTPRoute struct {
-	// Optional service type for Kubernetes solver service
-	// +optional
+	// Optional service type for Kubernetes solver service. Supported values
+	// are NodePort or ClusterIP. If unset, defaults to NodePort.
```

Comment:
question: why did we lose `+optional` here? not saying it's right or wrong but it stands out. Just to match the internal type with the external types?

Reply:
I don't know if these tags actually do anything at all... but I'll put it back.